submissions/boladeMarindoti/html/about.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+</head>
+<body>
+    <img src="./images/bells logo.png" id="logo" alt="bells logo">
+    <h1>About</h1>
+    
+        <uL>
+         <li><a href="index.html">Home</a></li>
```

Comment:
Linking of the pages in the navigation bar is your task for [8]. Remove them from all your pages and use `#` instead.